src/CliqueStateMachine.jl

```diff
@@ -339,6 +344,24 @@ function solveUp_StateMachine(csmc::CliqStateMachineContainer)
   return postUpSolve_StateMachine
 end
 
+"""
+  $SIGNATURES
+
+Notes
+- State machine function nr. #XXX
+"""
+function tryDownSolveOnly_StateMachine(csmc::CliqStateMachineContainer)
+  logCSM(csmc, "tryDownSolveOnly_StateMachine clique $(csmc.cliqKey) status $(getCliqueStatus(csmc.cliq))")
+
+  logCSM(csmc, "Skipping upsolve clique $(csmc.cliqKey)"; loglevel=Logging.Warn, st=getCliqueStatus(csmc.cliq))
+  if getCliqueStatus(csmc.cliq) == NULL 
+    logCSM(csmc, "Clique $(csmc.cliqKey) status NULL, trying as UPSOLVED"; loglevel=Logging.Warn)
+    setCliqueStatus!(csmc.cliq, UPSOLVED)
```

Comment:
Not sure skipping upsolve should set the status UPSOLVED, especially coming from NULL.  Trying to think what we can do better here...  Perhaps RECYCLED or SKIPPED is better?

Reply:
yes, I had the same worry, that's why the warning. I think you are correct with RECYCLED. Also, it needs some checks to see if it can be recycled.

Reply:
@dehann, see changes. I'm going with `solvedCount` > 0 then `UPRECYCLED` else `@error`